trunk/SUAVE/Methods/Weights/Correlations/BWB/payload.py

```diff
@@ -0,0 +1,55 @@
+# payload.py
```

Comment:
If they're common, let's make a common folder. We do this in other methods like in missions:

https://github.com/suavecode/SUAVE/tree/develop/trunk/SUAVE/Methods/Missions/Segments

If they need to be different then make a separate file. That way if there's a bug there's only one bug and not multiple of the exact same.